tests/acceptance/bootstrap/CliContext.php

```diff
@@ -496,4 +505,157 @@ public function theAdministratorTriggersEmailNotificationsUsingTheCLI(string $in
 
 		$this->featureContext->setResponse(CliHelper::runCommand($body));
 	}
+
+	/**
+	 * @Given the administrator has created stale upload
+	 *
+	 * @return void
+	 */
+	public function theAdministratorHasCreatedStaleUpload(): void {
+		$folderPath = $this->featureContext->getStorageUsersRoot() . "/uploads";
+		$infoFiles = glob($folderPath . '/*.info');
+		foreach ($infoFiles as $file) {
+			if (!unlink($file)) {
+				Assert::fail("Fail to delete info file");
+			}
+		}
+	}
+
+	/**
+	 * Runs the stale uploads CLI command
+	 *
+	 * @param bool $dryRun
+	 * @param string|null $spaceId
+	 *
+	 * @return ResponseInterface
+	 */
+	protected function runDeleteStaleUploadsCommand(bool $dryRun=true, ?string $spaceId = null): ResponseInterface {
+		$command = "storage-users uploads delete-stale-nodes";
+		$command .= " --dry-run=" . ($dryRun ? "true" : "false");
+
+		if ($spaceId !== null) {
+			$command .= " --spaceid=$spaceId";
+		}
+
+		$body = [
+			"command" => $command
+		];
+		return CliHelper::runCommand($body);
+	}
+
+	/**
+	 * @When the administrator lists all the stale uploads
+	 *
+	 * @return void
+	 */
+	public function theAdministratorListsAllTheStaleUploads(): void {
+		$this->featureContext->setResponse($this->runDeleteStaleUploadsCommand(true));
+	}
```

Comment:
```suggestion
	public function theAdministratorListsAllTheStaleUploads(): void {
		$this->featureContext->setResponse($this->runListOrDeleteStaleUploadsCommand(true));
	}
```
I think renaming the function will make less confusing.